docker/client.py

```diff
@@ -747,6 +747,8 @@ def inspect_container(self, container):
 
     @check_resource
     def inspect_image(self, image):
+        if isinstance(image, dict):
+            image = image.get('Id')
         return self._result(
```

Comment:
This isn't required to fix #602, but it's there to make this method more consistent with `inspect_container`.